services-custom/s3-transfer-manager/src/it/java/software/amazon/awssdk/transfer/s3/S3IntegrationTestBase.java

```diff
@@ -57,6 +58,7 @@
      */
     @BeforeClass
     public static void setUp() throws Exception {
+        Log.initLoggingToStdout(Log.LogLevel.Warn);
```

Comment:
I actually prefer to leave it here because it's very handy for troubleshooting. I checked the logs in the tests and warn didn't produce any logs for successful requests.